### PR TITLE
Release 13.2.0

### PR DESCRIPTION
**Pull request by r0mant**
Pending some Assist tests, do not merge yet.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+* Teleport Assist
+  * Added command execution progress feedback and settings UI. [#28480](https://github.com/gravitational/teleport/pull/28480)
+* Server Access
+  * Added option to preserve automatically created host users instead of deleting them. [#28432](https://github.com/gravitational/teleport/pull/28432)
+  * Fixed issue with `tsh join` when per-session MFA is enabled. [#28456](https://github.com/gravitational/teleport/pull/28456)
+* Database Access
+  * Updated `tsh db connect` to prefer `mongosh` client. [#28668](https://github.com/gravitational/teleport/pull/28668)
+  * Fixed issue with database agents not respecting graceful shutdown. [#28369](https://github.com/gravitational/teleport/pull/28369)
+* Device Trust
+  * Improved Jamf config validation. [#28553](https://github.com/gravitational/teleport/pull/28553)
```
Comment (zmb3) [inline CHANGELOG.md:14]: This is the first release that enables the Jamf integration, so can we just say something like:
```suggestion
  * Add Jamf integration for inventory management
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+  * Added Okta import rules support. [#28377](https://github.com/gravitational/teleport/pull/28377)
+  * Fixed issue with recreating a bot that was previously partially removed. [#28543](https://github.com/gravitational/teleport/pull/28543)
+* TLS Routing
+  * Added IP pinning support for TLS routing behind ALB mode. [#28466](https://github.com/gravitational/teleport/pull/28466)
+* Stability & Reliability
+  * Fixed issue with invalid database resources preventing cache initialization. [#28638](https://github.com/gravitational/teleport/pull/28638)
+* Web UI
+  * Added light & dark themes to YAML editor. [#28517](https://github.com/gravitational/teleport/pull/28517)
+  * Added light & dark themes to web terminal. [#28408](https://github.com/gravitational/teleport/pull/28408)
+
```
Comment (zmb3) [inline CHANGELOG.md:25]: There's a couple items on our upcoming releases page that arent here:

- Light theme in Teleport Connect
- Label expressions

**Review by zmb3 — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (r0mant) [reply CHANGELOG.md:25]: Those were already released I believe but we can mention them here again.

**Review by jakule — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (r0mant) [reply CHANGELOG.md:31]: Added.

**Pull request merged**